scripts/zones/Arrapago_Remnants/mobs/Archaic_Gear.lua

```diff
@@ -0,0 +1,16 @@
+-----------------------------------
+-- Area: Archaic Gear
+--  MOB: Arrapago Remnants
+-----------------------------------
+
+function onMobDeath(mob, player, isKiller)
```

Comment:
This duplicated behavior between gear and gears is probably fine for now but we may want to move this into either a mixin for gear(s) or something in a salvage.lua later.

Reply:
I was hesitant to start throwing globals into this since it was the first zone I attempted and wasn't sure what i could globalize. Progress tracking isn't something I'd want in a global file since each zone has different conditions to meet before progress gets changed.